examples/get_log_info.rs

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use openapi::apis::{configuration::Configuration, tlog_api};
+use openapi::models::LogInfo;
+
+#[tokio::main]
+async fn main() {
+    let configuration = Configuration::default();
+    let log_info: LogInfo = tlog_api::get_log_info(&configuration).await.unwrap();
+    println!("{:#?}", log_info);
+}
```

Comment:
I got this error (I think it ran correctly the first few times, though?):
```
[lily@localhost rekor-rs]$ cargo run --example get_log_info
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/get_log_info`
"rootHash: 59ce3532657a4d267161abf5dbba57fb6a71383198c7702bfa2e4d2b6e6fcd71\nsignedTreeHead: |\n  Rekor\n  2449157\n  Wc41MmV6TSZxYav127pX+2pxODGYx3Ar+i5NK25vzXE=\n  Timestamp: 1653505424656916632\n\n  — rekor.sigstore.dev wNI9ajBEAiB9guM7tvmpKSpEY+unOTEmRFUydRgm02qW0iSkQD/i8wIgXHFG+n6dZEAqxaN6yJjQdiv2utcuiFYv7gBFZK1vv88=\ntreeSize: 2449157\n"
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Serde(Error("expected value", line: 1, column: 1))', examples/get_log_info.rs:22:74
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Yeah I noticed this too.
I am not entirely sure why this is happening :thinking: 
![get_log_info_error](https://user-images.githubusercontent.com/60219242/171264162-9e9f4cdd-8c74-47b6-b13e-a9ca899cd853.png)